ext/net/ops.rs

```diff
@@ -53,6 +53,8 @@ pub fn init<P: NetPermissions + 'static>() -> Vec<OpPair> {
     ("op_net_listen", op_sync(op_net_listen::<P>)),
     ("op_dgram_recv", op_async(op_dgram_recv)),
     ("op_dgram_send", op_async(op_dgram_send::<P>)),
+    ("op_udp_setbroadcast", op_sync(op_udp_setbroadcast)),
+    ("op_udp_getbroadcast", op_sync(op_udp_getbroadcast)),
```

Comment:
For consistency with other getters/setters:
```suggestion
    ("op_udp_set_broadcast", op_sync(op_udp_set_broadcast)),
    ("op_udp_get_broadcast", op_sync(op_udp_get_broadcast)),
```